src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt

```diff
@@ -39,6 +40,12 @@ fun Application.apiModule(
         azureOpenidConfigTokenEndpoint = environment.azureOpenidConfigTokenEndpoint,
     )
 
+    val eregClient = EregClient(
+        azureAdClient = azureAdClient,
+        isproxyClientId = environment.isproxyClientId,
+        baseUrl = environment.isproxyUrl,
+    )
+
```

Comment:
Nå har jeg bare laget logikk for å hente informasjon fra Ereg slik at man kan få ut navn på virksomhet.
Planen er å tilby virksomhetsnavn i API slik at konsument slipper å hente dette selv og at dette gjøres asynkront i Isnarmestelder. Dvs. at jeg tenker å lage en Cronjob som kjører i bakgrunnen og legger til virksomhetsnavn i hver rad i tabellen i database. Grunnen til det er at jeg tenker det vil gi bedre ytelse å gjøre et slik i stedet for synkront med REST-kall. Virksomhetsnavn skal også ikke endres så det burde holde å hente dette en gang pers rad.